pdp/static/js/canada_ex_map.js

```diff
@@ -23,16 +31,24 @@ function init_raster_map() {
 
     na_osm = getNaBaseLayer(pdp.tilecache_url, 'North America OpenStreetMap', 'world_4326_osm', mapControls.projection);
 
-    defaults = {
-        dataset: "pr-tasmax-tasmin_day_BCSD-ANUSPLIN300-CanESM2_historical-rcp26_r1i1p1_19500101-21001231",
-        variable: "tasmax"
-    };
+    defaults = {};
+    if (archivePortal) {
+      //the old dataset
+      defaults.variable = "tasmax";
+      defaults.dataset = "pr-tasmax-tasmin_day_BCSD-ANUSPLIN300-CanESM2_historical-rcp26_r1i1p1_19500101-21001231";
+      defaults.timestamp = "2000-01-01";
+    } else {
+      //the new dataset
+      defaults.variable = "pr";
```

Comment:
Oops, I used pr/ACCESS0 because it was the first alphabetical file, so it was indexed and available first when I wanted to start testing while indexing was dragging on for twelve hours a file. I'll fix it.

-------

There _are_ trace negative values in the data:

```sql
SELECT MIN(data_file_variables.range_min) 
FROM 
  pcic_meta.ensembles, 
  pcic_meta.ensemble_data_file_variables, 
  pcic_meta.data_file_variables, 
  pcic_meta.variable_aliases
WHERE 
  ensemble_data_file_variables.ensemble_id = ensembles.ensemble_id AND
  ensemble_data_file_variables.data_file_variable_id = data_file_variables.data_file_variable_id AND
  data_file_variables.variable_alias_id = variable_aliases.variable_alias_id AND
  ensembles.ensemble_name = 'bccaq_version_2' AND 
  variable_aliases.variable_long_name = 'Precipitation';
```
`=> -8.13e-05`

We dealt with this issue previously when we added the gridded observation data: wanting to display pr with a log scale, even though it had data less than 0. What I did on gridded observation was to set a lower limit on COLORSCALERANGE of 1. This version of ncWMS displays all values lower than 1 as black, and ignored any supplied BELOWMINCOLOR parameters, so I also created a custom palette for ncWMS, which is described [here](https://pcic.uvic.ca/confluence/display/CSG/ncWMS-PCIC). Its lowest value is black, to blend seamlessly with the apparently-inescapable-black. So that should work again, and will take a lot less long to figure out this time. :)